packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -24,22 +23,11 @@ export const SidePanel = ({
 }: SidePanelProps) => {
   const ANIMATION_DURATION = 300
```

Comment:
i would suggest removing this value because this is misleading
`duration-${ANIMATION_DURATION}` does not accept arbitrary values and is expecting a predefined duration value in tailwind's set of duration class names, so i'd rather use `duration-300` directly in the classes

otherwise putting something like `duration-450` would result in an invalid class

Reply:
This is not used anymore, I will remove